packages/survey/src/Question/Item/style/index.ts

```diff
@@ -0,0 +1,49 @@
+import { CSSObject, useColorModeValue, useTheme } from '@chakra-ui/react'
+import { darken, isDark, lighten, transparentize } from '@chakra-ui/theme-tools'
+
+import { ItemProps } from '../models/ItemProps'
+
+export const itemStyle = ({ isChecked, activeColor, withImages }: ItemProps): CSSObject => {
+  const theme = useTheme()
+  const bgColor = useColorModeValue(
+    transparentize('black', 0.12)(theme),
+    transparentize('white', 0.29)(theme)
+  )
+  const hoverColor = useColorModeValue(
+    transparentize('black', 0.25)(theme),
+    transparentize('white', 0.22)(theme)
+  )
+  const hoverActiveColor = isDark(activeColor)(theme)
+    ? darken(activeColor, 0.05)(theme)
+    : lighten(activeColor, 0.05)(theme)
```

Comment:
```suggestion
    ? darken(activeColor, 5)(theme)
    : lighten(activeColor, 5)(theme)
```

It seems Chakra has a bug in his JSDoc for the `amounts` of the `lighten` and `darken` functions.

For example, they use in their [theme-tools](https://github.com/chakra-ui/chakra-ui/blob/main/packages/theme-tools/src/color.ts)

```ts
import Color from "tinycolor2"

/**
 * Lighten a specified color
 * @param color - the color in hex, rgb, or hsl
 * @param amount - the amount to lighten (0-1)
 */
export const lighten = (color: string, amount: number) => (theme: Dict) =>
  Color(getColor(theme, color)).lighten(amount).toHexString()
```

But [tinycolor](https://github.com/bgrins/TinyColor#lighten) says in its documentation

> ### lighten
> `lighten: function(amount = 10) -> TinyColor`. Lighten the color a given amount, from 0 to 100. Providing 100 will always return white.
> 
> ```ts
> tinycolor("#f00").lighten().toString(); // "#ff3333"
> tinycolor("#f00").lighten(100).toString(); // "#ffffff"
> ```

If I understand it correctly, the amount should be from `0 to 100` and not - like described by Chakra - `0 to 1`.

Reply:
Can you confirm this?
Does it look more like intentional with a value of `5` or does everything fit with your values of `0.05`?

Reply:
I also got a strange behavior:
when active color is dark should be lighter on hover and vice-versa when light. so I've changed the logic.
The color change on hover for some colors is really hard to see. Maybe you got a workaround?